Makefile

```diff
@@ -1,6 +1,6 @@
 REPO=fyndiq
 NAME=python-confluent-kafka
-TAG=1.5.0-python3.8.5-buster-v2
+TAG=1.5.0-python3.8.9-buster-v2
```

Comment:
Please name the tag `1.5.0-python3.8.9-buster-v1` instead of -v2. We should also consider upgrading the minor version, not sure if confluent-kafka-python have support (wheels) for this yet thou